lib/streamlit/elements/arrow_altair.py

```diff
@@ -250,6 +267,8 @@ def _arrow_bar_chart(
             If True, set the chart width to the column width. This takes
             precedence over the width argument.
             This argument can only be supplied by keyword.
+        theme : str or None
+            The theme of the chart. This argument only accepts "streamlit" or None for now.
```

Comment:
Maybe we can make the docstring description a bit more explanatory, e.g.:

`The theme of the chart. Currently, we only support "streamlit" for the Streamlit defined design or None to fallback to the default behavior of the library.`

Reply:
Btw. we need to add the theme parameter and the docstrings also in all of the chart method in:

https://github.com/streamlit/streamlit/blob/develop/lib/streamlit/elements/dataframe_selector.py